ezyrb/ann.py

```diff
@@ -9,27 +9,28 @@
 class ANN(Approximation):
     """
     Feed-Forward Artifical Neural Network (ANN).
+    : param int trained_epoch: number of already trained iterations.
+    : param criterion: Loss definition (Mean Squared). 
+    : type criterion: torch.nn.modules.loss.MSELoss.
     
     Example:
-    >>> import ezyrb
-    >>> import torch
-    >>> import torch.nn as nn
     >>> import numpy as np
-    >>> 
-    >>> x = np.random.uniform(-1, 1, size=(4, 2))
+    >>> from ann import ANN
```

Comment:
Not tested, but it looks wrong. Have you check the example?

Reply:
Yes, I checked in the Prove.pdf file I attached in my "273cad3" commit. I report a demonstration below. (Out[14] is the training loss at termination).
![Schermata 2021-02-23 alle 15 43 31](https://user-images.githubusercontent.com/75453926/108859868-f13df300-75ed-11eb-85ff-4fd01a0efa26.png)

Reply:
Hope that the following convince you: fresh installation of your commit, and tried the import you said be correct.
Take into account that testing the code means for all (or at least for the large majority) the environments, not only yours (or your directory, as in this case). Also please, check carefully the code, especially after someone's telling you there is probably a mistake. 

```
(base) ~ [ python3 -m pip uninstall ezyrb
Found existing installation: ezyrb 1.0
Uninstalling ezyrb-1.0:
  Would remove:
    /data/local/lib/python3.8/site-packages/ezyrb-1.0-py3.8.egg
Proceed (y/n)? y
  Successfully uninstalled ezyrb-1.0

(base) ~ [ python3 -m pip install git+git://github.com/ggurioli/EZyRB.git@012ee67c4a05d8b26153b633f78e96a868258d16
Defaulting to user installation because normal site-packages is not writeable
Collecting git+git://github.com/ggurioli/EZyRB.git@012ee67c4a05d8b26153b633f78e96a868258d16
  Cloning git://github.com/ggurioli/EZyRB.git (to revision 012ee67c4a05d8b26153b633f78e96a868258d16) to /tmp/pip-req-build-ag4g3xrr
  Running command git clone -q git://github.com/ggurioli/EZyRB.git /tmp/pip-req-build-ag4g3xrr
Requirement already satisfied: future in /data/local/lib/python3.8/site-packages (from ezyrb==1.1) (0.18.2)
Requirement already satisfied: numpy in /usr/lib/python3.8/site-packages (from ezyrb==1.1) (1.18.4)
Requirement already satisfied: scipy in /usr/lib/python3.8/site-packages (from ezyrb==1.1) (1.4.1)
Requirement already satisfied: matplotlib in /usr/lib/python3.8/site-packages (from ezyrb==1.1) (3.2.1)
Requirement already satisfied: GPy in /data/local/lib/python3.8/site-packages (from ezyrb==1.1) (1.9.9)
Requirement already satisfied: cycler>=0.10 in /usr/lib/python3.8/site-packages (from matplotlib->ezyrb==1.1) (0.10.0)
Requirement already satisfied: kiwisolver>=1.0.1 in /usr/lib/python3.8/site-packages (from matplotlib->ezyrb==1.1) (1.1.0)
Requirement already satisfied: pyparsing!=2.0.4,!=2.1.2,!=2.1.6,>=2.0.1 in /usr/lib/python3.8/site-packages (from matplotlib->ezyrb==1.1) (2.4.7)
Requirement already satisfied: python-dateutil>=2.1 in /usr/lib/python3.8/site-packages (from matplotlib->ezyrb==1.1) (2.8.1)
Requirement already satisfied: paramz>=0.9.0 in /data/local/lib/python3.8/site-packages (from GPy->ezyrb==1.1) (0.9.5)
Requirement already satisfied: six in /usr/lib/python3.8/site-packages (from GPy->ezyrb==1.1) (1.14.0)
Requirement already satisfied: setuptools in /data/local/lib/python3.8/site-packages (from kiwisolver>=1.0.1->matplotlib->ezyrb==1.1) (46.1.3)
Requirement already satisfied: decorator>=4.0.10 in /usr/lib/python3.8/site-packages (from paramz>=0.9.0->GPy->ezyrb==1.1) (4.4.2)
Building wheels for collected packages: ezyrb
  Building wheel for ezyrb (setup.py) ... done
  Created wheel for ezyrb: filename=ezyrb-1.1-py3-none-any.whl size=12613 sha256=8efae9831e3b19c55bbd161a3b43e525307cab3882d2d6dd19bd014abc5e63a4
  Stored in directory: /data/.cache/pip/wheels/f3/91/7b/d288de7188c19541e855435b77de759352c0c3575ff48e7b92
Successfully built ezyrb
Installing collected packages: ezyrb
Successfully installed ezyrb-1.1

(base) ~ [ python3
Python 3.8.2 (default, Apr  8 2020, 14:31:25) 
[GCC 9.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import ezyrb
>>> from ann import ANN
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'ann'

```

Reply:
Ok, now I understand what you mean. I've modified the __init__.py, installed my last commit, and tested the example as reported below. Is it fine?
![Init](https://user-images.githubusercontent.com/75453926/108869946-f9029500-75f7-11eb-9b67-d6b4e3e4f986.png)
![Terminal](https://user-images.githubusercontent.com/75453926/108869953-facc5880-75f7-11eb-9341-7a6662bc0f29.png)
![Test](https://user-images.githubusercontent.com/75453926/108869959-fb64ef00-75f7-11eb-81e4-faf90eb055c3.png)

Reply:
Now it seems reasonable!